aries-site/src/pages/components/card.mdx

```diff
@@ -51,16 +59,86 @@ A card by nature has limited real-estate, so it should present summarized inform
 
 ## When to use
 
+- Presenting high-level, summarized pieces of information.
+- Presenting an item, or collection of items, which can best be consumed visually.
+
+In all cases the presentation allows users to quickly scan the content and navigate to underlying detail if desired.
+
 ## When not to use
 
-- Do not use a Card to filter data on the same screen, instead use **\_**.
+- Cards should not be used as a control to filter data on the same screen. A button is the appropriate control to apply a filter. Alternatively, cards may be used present summarized information and allow a user to navigate to a pre-filtered view of that information.
+
+<BestPracticeGroup>
+  <Example
```

Comment:
<img width="1236" alt="Screen Shot 2022-09-27 at 1 03 25 PM" src="https://user-images.githubusercontent.com/54560994/192613602-97825253-14f6-41c8-b807-b54f85760504.png">
These two examples have slightly different heights, visually it might look a little cleaner if the do and don't red/green bars were vertically aligned with each other.

Also the search icon in the search input looks off.

Reply:
Search icon is awkward with scaling -- I'll have to look into that.

Fixed the difference in height.